hazelcast-client-legacy/src/main/java/com/hazelcast/client/proxy/ClientRingbufferProxy.java

```diff
@@ -103,7 +102,7 @@ public long remainingCapacity() {
     public long add(E item) {
         checkNotNull(item, "item can't be null");
 
-        AddRequest request = new AddRequest(getName(), toData(item));
+        AddRequest request = new AddRequest(getName(), toData(item), OverflowPolicy.OVERWRITE);
```

Comment:
This looks like change in the behaviour. Before in prepareOperation method OverflowPolicy.FAIL was used , now it looks like we are using  OverflowPolicy.OVERWRITE .